HomeTask3/Program.cs

```diff
@@ -0,0 +1,129 @@
+﻿var task = new MyHomework();
+//task.RunTask2();
+
+while (task.RunTask1())
+{
+
+}
+
+while (task.RunTask2())
+{
+
+}
+
+while (task.RunTask3())
+{
+
+}
+
+public class MyHomework
+{
+
+    public bool RunTask1()
+    {
+        Console.WriteLine("Enter the number:");
+        var numberFromConsole = Console.ReadLine();
+
+        if (numberFromConsole.ToLower() == "exit")
```

Comment:
Why such a specific check?

Reply:
I added more imagination and add the code in the loop which never ends. The entered from console word exit means that the program should break from loop.